### PR TITLE
fix: spotify name

### PR DESCRIPTION
**Pull request by mikejgray**
# Description
The name of the Raspotify device was not properly taking effect. This was due to a combination of the PHAL plugin not restarting the service as well as the properties in this skill not being set correctly.

# Issues
https://github.com/NeonGeckoCom/skill-mark2-audio-receiver/issues/15

### CODE REVIEW

**Comment by mikejgray** [comment]
Failing test does not appear related to change

File: skill_markII_audio_receiver/__init__.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.renaming_airplay = False
         self.renaming_spotify = False
+        self.initialize()
+
+    def initialize(self):
```
Comment (NeonDaniel) [inline skill_markII_audio_receiver/__init__.py:52]: I believe everything here can just be appended to `__init__` to resolve the test failures. As of `ovos-workshop` 0.0.12 init should handle bus init in the `super()` call

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: skill_markII_audio_receiver/__init__.py [hunk 2]
```diff
@@ -49,6 +49,26 @@ def __init__(self, *args, **kwargs):
...
+        airplay_name = self.airplay_name
+        self.log.info("Initializing Mark II Audio Receiver Skill with settings:")
+        self.log.info(
+            f"Renaming Raspotify device to settings value of {self.raspotify_name}"
+        )
+        self.bus.emit(
+            Message(
+                "neon.phal.plugin.audio.receiver.set.raspotify.name",
+                {"name": self.raspotify_name},
```
Comment (NeonDaniel) [inline skill_markII_audio_receiver/__init__.py:61]: adding context `{"skill_id": self.skill_id}` should resolve the test failures here..

File: skill_markII_audio_receiver/__init__.py [hunk 3]
```diff
@@ -49,6 +49,26 @@ def __init__(self, *args, **kwargs):
...
+            )
+        )
+        self.log.info(f"Renaming Airplay device to settings value of {airplay_name}")
+        self.bus.emit(
+            Message(
+                "neon.phal.plugin.audio.receiver.set.uxplay.name",
+                {"name": airplay_name},
```
Comment (NeonDaniel) [inline skill_markII_audio_receiver/__init__.py:69]: adding context {"skill_id": self.skill_id} should resolve the test failures here

File: skill_markII_audio_receiver/__init__.py [hunk 4]
```diff
@@ -205,11 +241,21 @@ def rename_spotify_intent(self, _) -> None:
             try:
                 self.renaming_spotify = False
                 self.log.debug(f"Renaming Raspotify device to {new_name}")
-                self.bus.emit(Message("neon.phal.plugin.audio.receiver.set.raspotify.name", {"name": new_name}))
+                self.bus.emit(
+                    Message(
```
Comment (NeonDaniel) [inline skill_markII_audio_receiver/__init__.py]: `message.forward` should be used here to preserve context (session, skill_id, etc)
```python
message.forward("neon.phal.plugin/audio.receiver.set.raspotify.name", {"name": new_name})
```

File: skill_markII_audio_receiver/__init__.py [hunk 5]
```diff
@@ -224,21 +270,41 @@ def rename_spotify_intent(self, _) -> None:
...
         if get_service_status("bluetooth") is True:
-            timeout = self.get_bluetooth_timeout
-            self.speak_dialog("pairing", data={"service": "bluetooth", "timeout": timeout})
-            self.bus.emit(Message("neon.phal.plugin.audio.receiver.pair.bluetooth", {"timeout": timeout}))
+            timeout = self.bluetooth_timeout
+            self.speak_dialog(
+                "pairing", data={"service": "bluetooth", "timeout": timeout}
+            )
+            self.bus.emit(
+                Message(
```
Comment (NeonDaniel) [inline skill_markII_audio_receiver/__init__.py]: same as above, should use `message.forward`

File: skill_markII_audio_receiver/__init__.py [hunk 6]
```diff
@@ -224,21 +270,41 @@ def rename_spotify_intent(self, _) -> None:
...
         if get_service_status("kdeconnect") is True:
-            timeout = self.get_kdeconnect_timeout
-            self.speak_dialog("pairing", data={"service": "KDE connect", "timeout": timeout})
-            self.bus.emit(Message("neon.phal.plugin.audio.receiver.pair.kdeconnect", {"timeout": timeout}))
+            timeout = self.kdeconnect_timeout
+            self.speak_dialog(
+                "pairing", data={"service": "KDE connect", "timeout": timeout}
+            )
+            self.bus.emit(
+                Message(
```
Comment (NeonDaniel) [inline skill_markII_audio_receiver/__init__.py]: same as above, should use `message.forward`

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**